test/JDBC/expected/BABEL-1661.out

```diff
@@ -1,3 +1,214 @@
+-- psql
+Alter system set babelfishpg_tds.product_version = "9.4.3000.6";
+SELECT pg_reload_conf();
+GO
+~~ERROR (Code: 0)~~
+
+~~ERROR (Message: ERROR: Please enter a valid major version number between 11 and 15
+    Server SQLState: 0A000)~~
+
+
+Alter system set babelfishpg_tds.product_version = "15.400.3000.6";
+SELECT pg_reload_conf();
+GO
+~~ERROR (Code: 0)~~
+
+~~ERROR (Message: ERROR: Please enter a valid minor version number
+    Server SQLState: 0A000)~~
+
+
+Alter system set babelfishpg_tds.product_version = "15.4.66000.6";
+SELECT pg_reload_conf();
+GO
+~~ERROR (Code: 0)~~
+
+~~ERROR (Message: ERROR: Please enter a valid micro version number
+    Server SQLState: 0A000)~~
```

Comment:
All the above are only checking one inequality. Please check both inequalities, e.g., 
```
(11 > atoi(token) || atoi(token) > 15)
```